simpledotorg/simple-server
# New "Reports Overview" card order

**Story card:** **[ch5858](https://app.shortcut.com/simpledotorg/story/5858/new-reports-overview-card-order)**

## Because
WHO requested a new card order for the "Reports Overview" page that makes it easier for CVHOs to discuss a region's progress while also keeping the focus around BP Control.

## This addresses
Reorders the 4 main cards in the "Reports Overview" as follows:
1. BP controlled
2. Registrations
3. BP not controlled
4. Missed visits

### Desktop
![image](https://user-images.githubusercontent.com/16785131/141519896-a0ddfef3-b149-497e-bd29-1ea3913fe8c4.png)
### Mobile
![image](https://user-images.githubusercontent.com/16785131/141520256-12f345e0-f961-4596-8e90-a0e8e273f1fc.png)

## Test instructions
1. Open your browser and look at a State, District, Block, and Facility report
2. Make sure there are no visual issues with the card's layout for each report on mobile and desktop
